src/kotlinSessions/SessionThree.kt

```diff
@@ -0,0 +1,231 @@
+package kotlinSessions
+
+/**
+ * The main class is the entry point of your application.
+ */
+
+fun main() {
+    //---------------------------------------Arrays----------------------------------------------
+    // An Array : is a list of some items, the items can be of Any type.
+    // the items have an order starting from (0) to (size - 1) and we usually call it an index.
+    // to get a particular item from an array, we append the subscript notation '[]' to our variable name and write our corresponding index inside
+    // asserting that its within the array range (0, size - 1).
+    // to define an Array use the type Array<Type> which will define an array of a specific type
+    val studentNames: Array<String> = arrayOf("Maged", "Abdo")
+    studentNames.forEach {
+        println(it)
+    }
+    // another way to define arrays of fixed size.
+    // the block {} is called an initialize phase for the constructor, it creates default values for our array while initializing it using the indices
+    val students: Array<String> = Array(3) {
+        index: Int -> "Name"
+    }
```

Comment:
you can just type
```.kt
val students: Array<String> = Array(3){"name"}
```

Reply:
I know, but you should be able to show them the full version.

Reply:
The discussion is : we want to assign each item by their indexes to this return value.